select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java

```diff
@@ -58,7 +57,7 @@
 
 	private AjaxSettings ajax;
 	private String data;
-	private String tags;
+	private Boolean tags;
```

Comment:
Unless `null` is a required third value I don't see a reason to use **B**oolean.